db/blob/blob_file_builder.cc

```diff
@@ -35,14 +36,15 @@ BlobFileBuilder::BlobFileBuilder(
     const std::string& column_family_name, Env::IOPriority io_priority,
     Env::WriteLifeTimeHint write_hint,
     const std::shared_ptr<IOTracer>& io_tracer,
-    BlobFileCompletionCallback* blob_callback,
+    BlobFileCompletionCallback* blob_callback, const std::string& dbname,
```

Comment:
Sorry, what I meant earlier is that DB name could also be a member of `BlobFileCompletionCallback` (in addition to the listeners). We don't really need it in `BlobFileBuilder`